packages/office-ui-fabric-react/src/components/GroupedList/GroupSpacer.tsx

```diff
@@ -18,6 +19,10 @@ export const GroupSpacer = (props: IGroupSpacerProps) =>
         'ms-GroupSpacer',
         styles.root
       ) }
-      style={ { width: props.count * SPACER_WIDTH } }
+      style={ { width: props.count * getWidthUnit(props.widthUnit) } }
```

Comment:
can this just be props.count * (props.widthUnit || SPACER_WIDTH ). It's typed to be a number already. the below function seems unnecessary.

Reply:
The reason why I went with this is to handle the case where the user passes 0 (falsy value). Not sure if it's a valid scenario we would want to support though.

Reply:
test to see if '0' causes an issue at all, and if so, just check for that. seems less code then an entire function.

Reply:
otherwise, seems good